stant-album/ViewController.swift

```diff
@@ -10,34 +10,82 @@ import UIKit
 
 class ViewController: UIViewController {
     @IBOutlet weak var albumCoverImageView: UIImageView!
-    @IBOutlet weak var albumNameLabel: UILabel!
-    @IBOutlet weak var artistNameLabel: UILabel!
-    @IBOutlet weak var recordLabel: UILabel!
-    @IBOutlet weak var newAlbumButton: UIButton!
+    @IBOutlet weak var albumNameLabel:      UILabel!
+    @IBOutlet weak var artistNameLabel:     UILabel!
+    @IBOutlet weak var recordLabel:         UILabel!
+    @IBOutlet weak var newAlbumButton:      UIButton!
     
-    var albumGenerator = AlbumGenerator()
+    var albumGenerator            = AlbumGenerator()
+    let child = SpinnerViewController()
+    let gradient: CAGradientLayer = CAGradientLayer()
     
     override func viewDidLoad() {
         super.viewDidLoad()
+        self.view.layer.insertSublayer(gradient, at: 0)
+        
+        albumNameLabel.textColor          = UIColor.white
+        albumNameLabel.font               = UIFont.boldSystemFont(ofSize: 25)
```

Comment:
In this case, as is on blocks, you can ident looking only on is blocks, removing some spaces.

Reply:
Done.